docs/DeveloperGuide.md

```diff
@@ -1,17 +1,26 @@
 # Developer Guide
 
 ## Table of Contents
+* [Setting up, getting started](#setting-up)
 * [Design & Implementation](#design--implementation)
     * [UI component](#ui-component)
     * [Logic component](#logic-component)
     * [Model component](#model-component)
-* [Product scope](#product-scope)
-    * [Target user profile](#target-user-profile)
-    * [Value proposition](#value-proposition)
-* [User Stories](#user-stories)
-* [Non-Functional Requirements](#non-functional-requirements)
-* [Glossary](#glossary)
-* [Instructions for manual testing](#instructions)
+* [Appendix: Requirements](#appendix-requirements)
+    * [Product scope](#product-scope)
```

Comment:
You feel that no need to put the User Profile and Value Proposition to the ToC?

Reply:
Yep, just following the ToC for addressbook since their user proifle and value prop is both under product scope and only product scope is in ToC